docs/content/preview/yugabyte-voyager/migrate/assess-migration.md

```diff
@@ -145,7 +145,7 @@ For the most accurate migration assessment, the source database must be actively
 
     After generating the report, navigate to the **Migrations** tab in the YugabyteDB UI at <http://127.0.0.1:15433> to see the available migrations.
```

Comment:
referred to as the YugabyteDB UI here and there, which isn't strictly correct, unless we were to adopt that more generally. But for now, it's yugabyted